### PR TITLE
E-mail Classification

### PR DESCRIPTION
**Pull request by psyduck1203**
## E-mail Classification

- [X] LGM-SOC'21 Participant 
- [X] Contributor

Closes: #30 

### Describe the add-ons or changes you've made

I have started with simple Exploratory Data Analysis(EDA), looked for some null or duplicate values. Then split the training and testing dataset using sklearn. Then I implemented different models to classify the message as spam or not spam. Also, the confusion matrix is displayed.

## Checklist:
- [X] My code follows the guidelines of this project.
- [X] I have performed a self-review of my own code.
- [X] I have commented on my code, particularly wherever it was hard to understand.
- [X] I have made corresponding changes to the documentation.
- [X] My changes generate no new warnings.
- [X] I have added things that prove my fix is effective or that my feature works.
- [X] Any dependent changes have been merged and published in downstream modules.

## Screenshots (Accuracy)

| Naive Bayes (Multinominal) Training Set Accuracy  | Naive Bayes (Multinominal) Testing Set Accuracy |
|:--------:|:------:|
| MNB-accuracy-test | MNB-accuracy-train |

### CODE REVIEW

**Review by prathimacode-hub — APPROVED** [review]
Well done. Your PR is approved and merged. Keep shining. Wonderful improvement showcased. Kudos. 👌🎇✔@psyduck1203
